src/components/Global/EarlyUserModal/index.tsx

```
@@ -0,0 +1,61 @@
+
+                    <ShareButton
+                        generateText={() => Promise.resolve(generateInvitesShareText(inviteLink))}
+                        title="Share your invite link"
+                    >
+                        Share Invite link
+                    </ShareButton>
+                    <a
+                        className="text-sm text-grey-1 underline"
+                        href="https://docs.peanut.me/og-and-invites"
+                        target="_blank"
+                    >
```

Biome (2.1.2)

[error] Avoid using target="_blank" without rel="noopener" or rel="noreferrer".

Opening external links in new tabs without rel="noopener" is a security risk. See the explanation for more details.
Safe fix: Add the rel="noopener" attribute.


(lint/security/noBlankTarget)